cmd/embedded-cluster/install.go

```diff
@@ -564,60 +572,65 @@ var installCommand = &cli.Command{
 		return nil
 	},
 	Flags: []cli.Flag{
-		&cli.BoolFlag{
-			Name:  "no-prompt",
-			Usage: "Disable interactive prompts. Admin console password will be set to password.",
-			Value: false,
-		},
-		&cli.StringFlag{
-			Name:   "overrides",
-			Usage:  "File with an EmbeddedClusterConfig object to override the default configuration",
-			Hidden: true,
-		},
 		&cli.StringFlag{
-			Name:    "license",
-			Aliases: []string{"l"},
-			Usage:   "Path to the application license file",
-			Hidden:  false,
+			Name:   "admin-console-password",
+			Usage:  "Password for the Admin Console",
+			Hidden: false,
 		},
 		&cli.StringFlag{
 			Name:   "airgap-bundle",
-			Usage:  "Path to the airgap bundle. If set, the installation will be completed without internet access.",
+			Usage:  "Path to the air gap bundle. If set, the installation will complete without internet access.",
 			Hidden: true,
 		},
 		&cli.StringFlag{
 			Name:   "http-proxy",
-			Usage:  "HTTP proxy to use for the installation",
+			Usage:  "Proxy server to use for HTTP",
 			Hidden: false,
 		},
 		&cli.StringFlag{
 			Name:   "https-proxy",
-			Usage:  "HTTPS proxy to use for the installation",
+			Usage:  "Proxy server to use for HTTPS",
 			Hidden: false,
 		},
+		&cli.StringFlag{
+			Name:    "license",
+			Aliases: []string{"l"},
+			Usage:   "Path to the license file",
+			Hidden:  false,
+		},
+		&cli.BoolFlag{
+			Name:  "no-prompt",
+			Usage: "Disable interactive prompts. The Admin Console password will be set to password.",
+			Value: false,
+		},
 		&cli.StringFlag{
 			Name:   "no-proxy",
```

Comment:
should this flag be right below the http-proxy and https-proxy flags?

Reply:
I considered grouping those together but did it alphabetically, which seems fairly standard.